h1st/model/fuzzy/fuzzy_model.py

```diff
@@ -0,0 +1,48 @@
+import imp
+import logging
+from typing import Dict
+
+from h1st.model.rule_based_model import RuleBasedModel
+from h1st.model.fuzzy.fuzzy_variables import FuzzyVariables
+from h1st.model.fuzzy.fuzzy_rules import FuzzyRules
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class FuzzyModel(RuleBasedModel):
+    """
+    FuzzyModel is a RuleBasedModel. You can create FuzzyModel using 
+    build_model method of FuzzyModeler. FuzzyModel has a property called
+    rules and it contains the core ControlSystem from skfuzzy package. 
+    For more information, check out https://scikit-fuzzy.github.io/scikit-fuzzy/.
+
+    """
+    def __init__(self):
+        super().__init__()
+        self.fuzzy_variables: FuzzyVariables = None
```

Comment:
Pass the construction parameters into the call. Avoid setting them separately.

Reply:
Related to the above comments